demisto_sdk/tests/test_files/fake_integration_expected_test_playbook__all_brands.yml

```diff
@@ -19,7 +19,7 @@ tasks:
       '#none#':
       - '1'
     separatecontext: false
-    view: '{"position": {"x": 50, "y": 50}}'
+    view: '{"position":{"x":50,"y":50}}'
```

Comment:
why are all these changes here? auto-formatted?

Reply:
Its a kind of a problem in the unittest that I have ugly-solved.